BuildHelpers/Public/Get-PSModuleManifest.ps1

```diff
@@ -57,13 +59,13 @@
         # Look for properly organized modules
         $ProjectPaths = Get-ChildItem $Path -Directory | 
             ForEach-Object {
-                $ThisFolder = $_
-                $ExpectedManifest = Join-Path $ThisFolder.FullName "$($ThisFolder.Name).psd1"
-                If( Test-Path $ExpectedManifest)
-                {
-                    $ExpectedManifest
-                }
+            $ThisFolder = $_
```

Comment:
Is there an auto formatter at work?  Seeing some odd behavior (e.g. here I wouldn't expect foreach cmdlet to be in line with it's scriptblock content)?

Reply:
VS Code formatting.
I will fix this